### PR TITLE
test: enhance QA script

### PR DESCRIPTION
**Pull request by izaera**
Basically make sure that we run `yarn link` in every folder under `packages` so
that packages are available to `link-js-toolkit` afterwards.

See this [slack conversation](https://liferay.slack.com/archives/G3JBR21HA/p1600938473003700?thread_ts=1599865341.101300&cid=G3JBR21HA)

### CODE REVIEW

File: scripts/qa/index.js [hunk 1]
```diff
@@ -65,6 +67,23 @@ if (argv['install']) {
 		cwd: linkJsToolkitProjectDir,
 	});
 
+	fs.readdirSync(projectsDir).forEach(projectName => {
+		const projectDir = path.join(projectsDir, projectName);
+		const pkgJson = fs.readJsonSync(path.join(projectDir, 'package.json'));
+
+		if (pkgJson['bin']) {
+			Object.keys(pkgJson['bin']).forEach(bin =>
```
Comment (wincent) [inline scripts/qa/index.js:75]: `pkgJson.bin` #amirite?

**Review by wincent — APPROVED** [review]
lg2m

File: scripts/qa/index.js (hunk 1 above)
Comment (izaera) [reply scripts/qa/index.js:75]: Well, in my defense I can say that this is `trial and error` code (yes, :facepalm:) and I forgot to refactor it when finished :shrug: .

Comment (jbalsas) [reply scripts/qa/index.js:75]: > [...] in my defense I can say that this is `trial and error` code [...]

That's really what you want to say in your defense? 😂

Comment (izaera) [reply scripts/qa/index.js]: Not what I want to say. The only I can say...

Comment (jbalsas) [reply scripts/qa/index.js]: If only not saying anything was an option...

just-sayin

**Pull request merged**